vuln/SCHEMA.md

```diff
@@ -0,0 +1,42 @@
+# Schema for vulnerabilities
+
+| Key | Type | Description | Example |
+|-----|------|-------------|---------|
+|`id`|Integer > 0|Unique number identifying the record|349|
+|`created_at`|date-time|Record creation|"2017-05-19T22:45:59.16+00:00"|
+XXX do we need this, or is it just a mongo internal field?
+
+|`updated_at`|date-time|Record update|"2017-06-05T13:01:57.071+00:00"|
+XXX do we need this, or is it just a mongo internal field?
```

Comment:
These are date fields that are useful. I'd keep them but it's totally up to you.